src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java

```diff
@@ -75,8 +72,9 @@
 	private final float renderDistanceMultiplier;
 	private final int resolution;
 	private final float intervalSize;
+	private final Float fov;
```

Comment:
What units is FOV in? (degrees or radians) - this is important to document in all the places we have an FOV field.